UPGRADE-3.3.md

```diff
@@ -177,6 +177,11 @@ HttpKernel
    which will tell the Kernel to use the response code set on the event's
    response object.
 
+Intl
+----
+
+ * Rename `TimeZoneTransformer` class into `TimezoneTransformer`
```

Comment:
since the class is internal (since 2.8), I'd suggest removing these, and apply this PR on 3.2